node/src/components/block_accumulator.rs

```diff
@@ -323,7 +322,7 @@ impl BlockAccumulator {
             Some(acceptor) => acceptor,
         };
 
-        match acceptor.register_block(block, sender, executed) {
+        match acceptor.register_block(block, sender) {
```

Comment:
This reverts (at least partially) the changes introduced here: https://github.com/casper-network/casper-node/pull/3405

Are we sure that `put-deploy` still [works as expected](https://github.com/casper-network/casper-node/issues/3388)?